catalogs/catalog-messaging-kafka/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+description = "catalog-messaging-kafka"
+
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+dependencies {
+  implementation(project(":api"))
+  implementation(project(":core"))
+  implementation(project(":common"))
+}
+
+tasks {
+  val runtimeJars by registering(Copy::class) {
+    from(configurations.runtimeClasspath)
+    into("build/libs")
+  }
+
+  val copyCatalogLibs by registering(Copy::class) {
+    dependsOn(jar, runtimeJars)
+    from("build/libs")
+    into("$rootDir/distribution/package/catalogs/messaging-kafka/libs")
+  }
+
+  val copyCatalogConfig by registering(Copy::class) {
+    from("src/main/resources")
+    into("$rootDir/distribution/package/catalogs/messaging-kafka/conf")
+
+    // TODO. add configuration file later on.
+
+    rename { original ->
+      if (original.endsWith(".template")) {
+        original.replace(".template", "")
+      } else {
+        original
+      }
+    }
+
+    exclude { details ->
+      details.file.isDirectory()
+    }
+  }
+
+  register("copyLibAndConfig", Copy::class) {
+    dependsOn(copyCatalogConfig, copyCatalogLibs)
+  }
+}
```

Comment:
I think you still missing several integration related codes here, are you going to do this in a separate PR?

Reply:
yes, a todo added to the below